### DISCUSSION TITLE
Positioning is only relative to the parent

### DISCUSSION THREAD

**Discussion by schneedorfer** [floating-ui/floating-ui#2320] [React Native]
The library only computes relative position to the parent and not absolute page position. I need to portal the popover to root of the app, so the relative position is useless in that case. Is there a way I can use this library with the given example?

edit: I was wrong, the library does compute absolute position as well

**Comment by atomiks** [comment]
The measurements are made here: https://github.com/floating-ui/floating-ui/blob/88db064ccbefbdef20f6d9ab851d402ce06baaf1/packages/react-native/src/createPlatform.ts#L18-L48

If you specify the `offsetParent` where the floating element is portaled, it should work

**Reply by schneedorfer** [reply]
Actually what fixed the issue is setting `sameScrollView` to false, I noticed that in the code it then uses the `measureInWindow` method instead. Just from curiosity tho, is there a difference of how the measure methods work wheter or not the reference element is in a scroll view?

**Reply by ShivamJoker** [reply]
I am not sure why it is not the default behavior.
I spent more than an hour figuring this out.
